prefab/prefab-publishing/mylibrary/build.gradle

```diff
@@ -39,14 +39,27 @@ android {
         }
     }
 
+    // Enable creation of Prefab packages and integration into AAR.
     buildFeatures {
         prefabPublishing true
     }
 
+    // Configure prefab to include the "mylibrary" module from the native build system,
+    // and export the headers in src/main/cpp/include.
```

Comment:
Include the "mylibrary" module from the native build system in the AAR, and export the headers in src/main/cpp/include to its consumers.

Reply:
done